example-projects/game-processor/index.js

```diff
@@ -3,7 +3,7 @@ const mqtt = require('async-mqtt');
 const { v4: uuidv4 } = require('uuid');
 const MongoClient = require('mongodb').MongoClient;
 const Config = require('./config');
-var mongodbUrl = `mongodb://${Config.mongodbUsername}:${Config.mongodbPassword}@${Config.mongodbHost}`;
+var mongodbUrl = `mongodb://localhost:27017/processor`;
```

Comment:
Why change this from using configs?

Reply:
There is an issue here usually when sm1 install mongodb he doesnt not have a username or password so the example will not work in some cases. i will try to make sure it works without setting a user and password